docs/quick-start/create-near-app.md

```diff
@@ -12,7 +12,7 @@ Setup a NEAR Protocol web app template using one command, `npx create-near-app`.
 npx create-near-app my-awesome-project
 cd my-awesome-project
 yarn
```

Comment:
this one is optional now (is done automatically when project is created)

Reply:
✅